release_notes_generator/model/pull_request_record.py

```
@@ -0,0 +1,288 @@
+"""
+A module that defines the PullRequestRecord class, which represents a pull request record in the release notes.
+"""
+
+import re
+from typing import Optional, Any
+
+from github.Commit import Commit
+from github.Issue import Issue
```

Ruff (0.12.2)

`github.Issue.Issue` imported but unused

Remove unused import: `github.Issue.Issue`

(F401)